raylib/Cargo.toml

```diff
@@ -12,7 +12,7 @@ categories = ["api-bindings", "game-engines", "graphics"]
 edition = "2018"
 
 [dependencies]
-raylib-sys = { version = "3.7", path = "../raylib-sys" }
+raylib-sys = { version = "= 3.7.0", path = "../raylib-sys" }
```

Comment:
I'm assuming that adjusting the versioning in this place will change the versioning that would get uploaded to crates.io if there was, hypothetically a Raylib 3.8 or 3.9 release. But I'm not certain of that.